packages/config/config/devices/0x027a/zen31.json

```
@@ -256,12 +139,12 @@
 		},
 		{
 			"#": "151",
-			"label": "Ramp Rate (switch)",
-			"description": "How long it takes for the dimmer to reach full brightness or turn off when operated from the connected wall switch.",
+			"label": "Ramp Rate (Switch)",
+			"description": "1-127 = seconds, 128-254 = 1-127 minutes",
```

For consistency with other configurations I prefer
```suggestion
			"description": "1...127 = seconds, 128...254 = minutes (minus 127)",
```